source/guide_moodle.rst

```diff
@@ -0,0 +1,215 @@
+
+Moodledata
+----------
+Moodle saves files and data into a folder inter alia for persistency. We have to create this folder and give its path to Moodle later on.
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/moodledata
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+First, we're going to download Moodle and extract it. For this, we should first change into the html directory of your account.
+
+Then we're going to download the latest version (Moodle-Download_ page) of the tgz archive and extract it. I would advise you to select the stable version. Note that the downloadlink on the page is actually a redirect. So instead of the link behind the button, use the direct link e.g.: ``https://download.moodle.org/download.php/direct/...``.
+
+::
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust html]$ wget -O moodle.tgz https://download.moodle.org/download.php/direct/stable38/moodle-latest-38.tgz
+ [isabell@stardust html]$ tar --strip-components=1 --gzip --extract --file=moodle.tgz
+ [isabell@stardust html]$ rm moodle.tgz
+ [isabell@stardust html]$
+
+The ``--strip-components=1`` option will unpack the contents of the moodle folder inside the archive into the current folder. You could also unpack the archive with its subdirectory if you plan to use it with your own domain:
+
+::
+
+ [isabell@stardust html]$ tar --gzip --extract --file=moodle.tgz
+ [isabell@stardust html]$
+
+Step 2
+------
+Now we should be able to access the initialization process of the Moodle instance. Here, we used the uberspace domain, so our address is the one mentionend here:
+
+.. include:: includes/web-domain-list.rst
+
+We visit the page in the browser of our choice.
+
+1. Choose any language and click ``Next``
+2. For the ``Data directory``, we choose the moodledata folder we created. ``/home/isabell/moodledata``
+3. For the Database driver, we choose ``MariaDB (native/mariadb)``
+4. For the Database Settings, we enter our information:
+  
+  1. ``Database name``: ``isabell_moodle``
+  2. ``Database user``: ``isabell``
+  3. ``Database password``: ``MySuperSecretPassword``
+  4. ``Database port``: ``3306``
+
+5. Read the Copyright notice and only accept if you truly do :)
+6. The checks should now show all OK. At the end of the page, we click on ``Continue``. This could take a while to finish.
+7. You should be redirected to the installation status screen. If you're not being redirected automatically, just wait a little bit. Try refreshing the exact site. It has to be an URL something like https://isabell.uber.space/admin/index.php?cache=0&agreelicense=1&confirmrelease=1&lang=en . At this point, it may be possible that a message like "Site is being upgraded, please retry later" pops up after reloading. Just try reloading after some minutes when this occurs.
```

Comment:
While trying this guide I had major troubles with this. Even after several reloads and a bunch of waiting, moodle wouldn't show up. Once it finally did the 2nd stage of the setup process wouldn't continue until I deleted a database table it complained about. In the end it was working, but that's not something we can document and support for users.

We'll do a few performance upgrade in the following weeks. Once those are done, I'll take another look. Sorry about the delay.